src/compo/modis_aod2ioda.py

```diff
@@ -133,14 +156,20 @@ def _read(self):
             over_land = np.logical_not(land_sea_flag == 0)
             UNC = np.where(over_land, unc_land, np.add(0.05, np.multiply(0.15, aod)))
 
-            self.outdata[('latitude', metaDataName)] = np.append(self.outdata[('latitude', metaDataName)], np.array(lats, dtype=np.float32))
-            self.outdata[('longitude', metaDataName)] = np.append(self.outdata[('longitude', metaDataName)], np.array(lons, dtype=np.float32))
-            self.outdata[('dateTime', metaDataName)] = np.append(self.outdata[('dateTime', metaDataName)], np.array(obs_time, dtype=np.int64))
+            self.outdata[('latitude', metaDataName)] = np.append(self.outdata[('latitude', metaDataName)],
+                                                                 np.array(lats, dtype=np.float32))
```

Comment:
Do you need to apply `winmsk` to lat, lon, and dateTime too?

Reply:
Yeah! Thanks for catching it.